Orange/widgets/utils/scaling.py

```diff
@@ -47,7 +47,9 @@ def _compute_scaled_data(self):
             return
 
         Y = data.Y if data.Y.ndim == 2 else np.atleast_2d(data.Y).T
-        self.original_data = np.hstack((data.X, Y)).T
+        self.original_data = np.hstack((data.X if "csr_matrix" not in str(type(data.X))
+                                        else data.X.toarray(),
```

Comment:
We shouldn't transform sparse matrix to dense here but should rather adapt the method to work on sparse matrices.